src/app/view/components/dot-container-selector/dot-container-selector.component.ts

```diff
@@ -30,11 +30,12 @@ export class ContainerSelectorComponent implements OnInit {
     }
 
     /**
-     * Call when the selected site changed and the change event is emmited
-     * @param {*} container
-     * @memberof ContainerSelectorComponent
+     * Called when the selected site changed and the change event is emmited
+     *
+     * @param {DotContainer} container
```

Comment:
Sometimes your method change after the doc has been generated, make sure you have the doc always updated... also use "Document This" extension.